### ISSUE TITLE
Simple checkout flow fails when 3ds verification fail

### ISSUE THREAD

**Issue by bulivlad** [VivaPayments/API#1821]
Simple checkout flow fails to call form action on demo account when the user selects 'Fail' on 'Choose your authentication outcome.' dropdown.

The https://demo.vivapayments.com/web/checkout/getpaymenttoken call fails with 403 error preventing form action to be executed

<img width="418" alt="Screenshot 2020-08-20 at 02 25 49" src="https://user-images.githubusercontent.com/6691306/90699663-83a2ba00-e28c-11ea-917c-601d0fdd9fcb.png">

<img width="615" alt="Screenshot 2020-08-20 at 02 24 03" src="https://user-images.githubusercontent.com/6691306/90699657-80a7c980-e28c-11ea-82b0-289581db0ce8.png">

**Comment by matthewrgourd** [comment]
Hi @bulivlad, sorry to hear this. We're aware there's an issue and working on a fix. I will update you once it is done. 
Duplicate of 1820#

**Comment by matthewrgourd** [comment]
Hi @bulivlad, I'm advised that a fix was applied last week. Please close this issue if problem is now solved.

**Comment by bulivlad** [comment]
Hi @matthewrgourd, it seems to be fixed. Thanks
